src/screens/GalleryScreen/GalleryScreen.tsx

```diff
@@ -117,27 +119,28 @@ export const GalleryScreen: React.FC = () => {
   const getPhoto = async () => {
     try {
       const photoArrayFromLocalStorage = await AsyncStorage.getItem(photoKey);
+      //Guard
       if (photoArrayFromLocalStorage === null) {
         Alert.alert('No photos from local storage');
-      } else {
-        const photoArrayJSON: PhotoFile[] = JSON.parse(
-          photoArrayFromLocalStorage,
-        );
-        setPhotoArrayState(photoArrayJSON);
-        console.log('Got photo! ');
+        return;
       }
+
+      const photoArrayJSON: PhotoFile[] = JSON.parse(
+        photoArrayFromLocalStorage,
+      );
+
+      setPhotoArrayState(photoArrayJSON);
     } catch (error) {
       Alert.alert('Something went wrong at GalleryScreen PhotoJSON');
-      console.warn(error);
+      console.error(error);
     }
   };
 
-  const renderPhoto = ({item}) => (
+  const renderPhoto = ({item}: any) => (
```

Comment:
Same here like the previous comment. We should know the type and our complier should be able to auto complete the fields.